images/ose-must-gather.yml

```diff
@@ -20,7 +20,7 @@ for_payload: true
 from:
   builder:
   - stream: golang
-  member: openshift-enterprise-cli
+  member: ose-cli-artifacts
```

Comment:
Do we still need this change considering that must-gather still bases on `cli` image and only extract oc.rhel8 from cli-artifacts in https://github.com/openshift/must-gather/pull/418